src/pybind/mgr/dashboard/frontend/package.json

```diff
@@ -26,11 +26,13 @@
     "lint:prettier": "prettier --list-different \"{src,cypress}/**/*.{ts,scss}\"",
     "lint:html": "htmllint src/app/**/*.html && html-linter --config html-linter.config.json",
     "lint:tsc": "tsc -p src/tsconfig.app.json --noEmit && tsc -p tsconfig.spec.json --noEmit && tsc -p cypress/tsconfig.json --noEmit",
-    "lint": "npm run lint:tsc && npm run lint:tslint && npm run lint:prettier && npm run lint:html",
+    "lint:scss": "stylelint '**/*.scss'",
+    "lint": "npm run lint:tsc && npm run lint:tslint && npm run lint:prettier && npm run lint:html && npm run lint:scss",
```

Comment:
I reiterate this suggestion: https://github.com/ceph/ceph/pull/35225#discussion_r433164558

Reply:
done.